.travis.yml

```diff
@@ -1,16 +1,14 @@
 language: php
 
 php:
-  - 5.4
-  - 5.5
   - 5.6
   - 7.0
   - 7.1
   - hhvm
```

Comment:
Feel free to drop HHVM as well if it causes you any trouble.

Reply:
Waiting for `Travis CI`'s answer :smile: